doc/source/indexing.rst

```diff
@@ -1307,7 +1312,7 @@ The ``in`` and ``not in`` operators
 
    try:
        old_d = d
-       del d
+       del d                                                       # noqa: F821
```

Comment:
same as before, and I'm not sure it makes sense that `d` is not defined here to need the `noqa`, but it is in the previous line.